packages/hadron-document/src/element.ts

```diff
@@ -775,6 +818,17 @@ export class Element extends EventEmitter {
     return isPlainObject(value) || isArray(value);
   }
 
+  isExpanded(includeChildren = false): boolean {
```

Comment:
Actually just generally, is there ever a case where this nested check makes sense? The elemet is either expanded or not, whether or not it's children are expanded should be of no difference for this check. What's the use-case for this?